projects/cdk/pipes/replace/replace.pipe.ts

```diff
@@ -0,0 +1,30 @@
+import {Pipe, PipeTransform} from '@angular/core';
+
+type SearchValue = RegExp | string;
```

Comment:
Without overloads let's not extract types.

Reply:
fixed